### DISCUSSION TITLE
href link

### DISCUSSION THREAD

**Discussion by melikst** [AutomaApp/automa#281] [Q&A] [answered]
Is there a way to get href link of element? Tried using XPath with `//a[@class='class_name']/@href` and it returns `undefined`

**Comment by melikst** [comment, marked as answer]
Haven't fully figured it out Automa functionality yet. Solved it with `Attribute value` and `href` as `Attribute name`.

**Reply by yosiasm** [reply]
hey @itocha I still don't understand how you solve it. Would you explain it more to us?

**Reply by Kholid060** [reply]
@yosiasm to get the URL from an [anchor element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a), use the [Attribute Value block](https://docs.automa.site/blocks/attribute-value.html) and in the `Attribute name` form input `href`.

![image](https://user-images.githubusercontent.com/22908993/152946243-0a5c19d5-b801-4154-a94d-d6a64be14f32.png)

**Reply by yosiasm** [reply]
What if the HTML look like `<div class="selectedClass"> <a href="IWantThisValue.com"> </div>`?

**Reply by Kholid060** [reply]
Well, you can use the same config as the image above but with `.selectedClass a` as the selector.

**Reply by yosiasm** [reply]
Great! It works.  Thank you.
